docs/where-to-go-for-help.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: default
+title: Where To Go for Help
+---
+
+# Where To Go for Help
+
+Per the *FIRST* Role Description, the AV volunteer reports to the Event Coordinator (EC), but you can get help for some things from other people at the event or on Slack.
+
+For technical questions with the AV cart or any software on it, talk to someone in the **#av-help channel on Slack**.
+
+For issues with Audience Display not showing the correct information or issues with FMS (the server that runs the field), talk to your ***FIRST* Technical Advisor (FTA)**.
+
+For questions about whether/where you can set up equipment (like cameras) near the field, talk to your **field supervisor**.
+
+For questions about AV which may affect match play, such as how audible game sounds are or changing lighting conditions, talk to your ***FIRST* Technical Advisor (FTA)**.
+
+For venue-specific questions, like where to plug in ethernet or sound, or where to put the projector, talk to your **Event Coordinator (EC)**.
+
+Feeling like you need an extra hand? Talk to your **Volunteer Coordinator (VC)**, they might have someone in mind who can help you out.
+
+For any other questions, reach out to the **#av-help channel on Slack** and we can point you in the right direction. Keep in mind that every event is a little different so the first person that you ask may not have all the answers, you can always fall back to Slack where a larger group will see your question and be able to chime in.
```

Comment:
same here, add link to av-help
https://first-in-michigan.slack.com/archives/C04PTSR77H7